doc/man1/openssl-s_client.pod.in

```diff
@@ -123,13 +154,25 @@ manual page.
 
 Print out a usage message.
 
+=item B<-ssl_config> I<filename>
+
+Use the specified configuration file.
+
```

Comment:
Could you please provide more information describing how this file interferes with the default config?

Reply:
I didn't know.  I did some code-reading.  It turns out this names the *section* of config to use, not the config file itself.  I pushed an additional commit to address this.